### PR TITLE
Tdrd 539 adding hsts header

### PR DESCRIPTION
**Pull request by Tom-Hallett**

### CODE REVIEW

File: test/test_signed_cookies.py [hunk 1]
```diff
@@ -80,6 +80,13 @@ def test_access_control_origin(environment, origin, allowed_origin, frontend_url
     response = signed_cookies.generate_response(cookies, environment, frontend_url, origin)
     assert allowed_origin == response["headers"]["Access-Control-Allow-Origin"]
 
+@pytest.mark.parametrize("environment, origin, allowed_origin, frontend_url", origin_test_values())
+def test_header_reponse_values(environment, origin, allowed_origin, frontend_url, kms, httpserver: HTTPServer):
```
Comment (TomJKing) [inline test/test_signed_cookies.py]: Spelling in function name: test_header_**response**_values

File: test/test_signed_cookies.py [hunk 2]
```diff
@@ -80,6 +80,13 @@ def test_access_control_origin(environment, origin, allowed_origin, frontend_url
...
+    set_up(kms, httpserver)
+    cookies = {"signature": "test_signature", "policy": "test_policy", "key_pair_id": "test_key_pair_id"}
+    response = signed_cookies.generate_response(cookies, environment, frontend_url, origin)
```
Comment (TomJKing) [inline test/test_signed_cookies.py:87]: Should test for the other header: `Access-Control-Allow-Origin`

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of very minor comments.

File: test/test_signed_cookies.py (hunk 2 above)
Comment (Tom-Hallett) [reply test/test_signed_cookies.py:87]: Other header is tested in a separate function above this one.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**